server/public/model/utils.go

```diff
@@ -555,16 +530,6 @@ func StringInterfaceFromJSON(data io.Reader) map[string]any {
 	return objmap
 }
 
-func StructFromJSONLimited[V any](data io.Reader, maxBytes int64, obj *V) error {
```

Comment:
I would prefer we kept a `StructFromJSON` function that all the methods used. That way we do all our deserialization in a common place and way.

Reply:
Done.